deep_speech_2/evaluate.py

```diff
@@ -111,6 +112,14 @@
     default='datasets/vocab/eng_vocab.txt',
     type=str,
     help="Vocabulary filepath. (default: %(default)s)")
+parser.add_argument(
+    "--error_rate_type",
+    default='wer',
+    choices=['wer', 'cer'],
+    type=str,
+    help="There are total two error rate types including wer and cer. wer "
+    "represents for word error rate while cer for character error rate. "
+    "(default: %(default)s)")
```

Comment:
Make the wording concise, e.g. `Error rate type for evaluation. 'wer' for word error rate and 'cer' for character error rate.`